core/audits/metrics/max-potential-fid.js

```diff
@@ -61,6 +108,11 @@ class MaxPotentialFID extends Audit {
       settings: context.settings, URL: artifacts.URL};
     const metricResult = await ComputedFid.request(metricComputationData, context);
 
+    const processedTrace = await ProcessedTrace.request(trace, context);
+    const processedNavigation = await ProcessedNavigation.request(trace, context);
+    const details = MaxPotentialFID.getLongAnimationFrameDetails(processedTrace,
```

Comment:
I'm fine with sticking this in debug data. What's the reasoning for putting it in MPFID as opposed to TBT though?

Reply:
Meh I think it's fine. TBT is just where my brain went since that's an official metric.